.sisyphus/notepads/twitch-adblock-enhancement/issues.md

```
@@ -0,0 +1,33 @@
+# Issues - Twitch Ad-Block Enhancement
+
+## 2026-01-11: Known Issues & Gotchas
+
+### Technical Gotchas
+
+1. **Client-Integrity Header**: May need periodic updates from VAFT project
+   - Monitor: https://github.com/pixeltris/TwitchAdSolutions/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)